### PR TITLE
[All] (NSD) Optional connected experiences behavior

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[cfa6764](https://github.com/OfficeDev/office-js-docs-pr/commits/cfa6764baf81f50806f8989c74af545d58b8fd32)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/privacy-and-security?branch=pr-en-us-2129) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2129) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008271737299201-2129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=5WXYA0v0da5Vfa%2BfdwuBTd4oHMDNc0wsRj9TLTxMIOU%3D&st=2020-08-27T17%3A33%3A19Z&se=2020-09-27T17%3A38%3A19Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/concepts/privacy-and-security.md [hunk 1]
```diff
@@ -69,6 +68,14 @@ You can make your Office Add-ins available to the public by publishing them to A
 
 - Supports a user-review system for available add-ins to promote a self-policing community.
 
+## Optional connected experiences
+
+Users can turn off [optional connected experiences in Office](/deployoffice/privacy/optional-connected-experiences) desktop and mobile clients. For Office Add-ins, generally this means that users who turn off the **Optional connected experiences** setting can no longer access add-ins or the Office Store through these clients. However, certain Microsoft add-ins that are considered essential or business-critical, and add-ins deployed by an organization's IT admin are still available. Also, the setting doesn't affect the availability of add-ins and the Store in Office on the web.
```
Comment (JuaneloJuanelo) [inline docs/concepts/privacy-and-security.md:73]: IT Admins and End users

Comment (JuaneloJuanelo) [inline docs/concepts/privacy-and-security.md:73]: generally? lets be more bold. The impact of turning OFF Optional connected experiences means.....

Comment (JuaneloJuanelo) [inline docs/concepts/privacy-and-security.md:73]: users and admins

Comment (JuaneloJuanelo) [inline docs/concepts/privacy-and-security.md:73]: add-ins deployed via [link] Centralized deployment by IT Admins.....

Comment (JuaneloJuanelo) [inline docs/concepts/privacy-and-security.md:73]: will still  be available

Comment (JuaneloJuanelo) [inline docs/concepts/privacy-and-security.md:73]: optional connected experiences DO affect Excel And Word Online, but not outlook (please check this page:   https://docs.microsoft.com/en-us/deployoffice/privacy/overview-privacy-controls. so i think this explanation is an outlook specific one.

File: docs/outlook/privacy-and-security.md [hunk 2]
```diff
@@ -42,19 +42,27 @@ The following figure shows the four levels of permissions and describes the capa
...
+- Requires a developer to provide proof of identity, a contractual agreement, and a compliant privacy policy to submit add-ins.
 
 - Archives add-ins in read-only mode.
 
 - Supports a user-review system for available add-ins to promote a self-policing community.
 
-## End users: privacy and performance concerns
+## Optional connected experiences
+
+Users can turn off [optional connected experiences in Office](/deployoffice/privacy/optional-connected-experiences) desktop and mobile clients. For Outlook add-ins, generally this means that users who turn off the **Optional connected experiences** setting can no longer get add-ins through these clients. However, certain Microsoft add-ins that are considered essential or business-critical, and add-ins deployed by an organization's IT admin are still available. Since the setting doesn't affect the availability of add-ins and the Office Store in Outlook on the web, the user can use the web client to [manage their add-ins](https://support.microsoft.com/office/8f2ce816-5df4-44a5-958c-f7f9d6dabdce), including admin-deployed ones.
```
Comment (JuaneloJuanelo) [inline docs/outlook/privacy-and-security.md:59]: should we mention that the get Add-ins button will NOT be available in Windows and MAc? but they can be managed in OWA.  In mobile they will have the get add-ins experience but just with add-ins deployed via centralized deployemnt (admin-managed)

**Comment by o365devx** [comment]
Docs Build status updates of commit _[76cf021](https://github.com/OfficeDev/office-js-docs-pr/commits/76cf0214039d3b9ee78f7eb91bae6d9afc4ff9af)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/privacy-and-security?branch=pr-en-us-2129) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2129) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008272247099457-2129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IdiTYU12PijVGZnxUcyGJKO7oan8hBIZ%2Bf1IMemRHtM%3D&st=2020-08-27T22%3A42%3A55Z&se=2020-09-27T22%3A47%3A55Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f26bcfc](https://github.com/OfficeDev/office-js-docs-pr/commits/f26bcfc15e063da95dbd6a6872cbaae8209bc048)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/privacy-and-security?branch=pr-en-us-2129) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2129) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202008272340456178-2129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=IDNzIFxuCV3dKl6nDBzTBoH7UfzxHF0lsU8u0tQEFVo%3D&st=2020-08-27T23%3A36%3A27Z&se=2020-09-27T23%3A41%3A27Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1ddd0fb](https://github.com/OfficeDev/office-js-docs-pr/commits/1ddd0fb2338f02ae0153c9f718af1236ba0bc496)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/privacy-and-security?branch=pr-en-us-2129) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2129) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C11%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009111601035507-2129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=2o5yqfg5%2FlGQkpe3ZKZCpVw77QdP4AzDXH7iXocE9iY%3D&st=2020-09-11T15%3A56%3A43Z&se=2020-10-12T16%3A01%3A43Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[961054a](https://github.com/OfficeDev/office-js-docs-pr/commits/961054a8221f9b4808478374bb6ebcc1f4b9b4e0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/privacy-and-security?branch=pr-en-us-2129) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2129) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009141612461208-2129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=yw7RuVFVR21D1vm38SMwQT8lz8QDplwirIp4uLH53ZA%3D&st=2020-09-14T16%3A08%3A32Z&se=2020-10-15T16%3A13%3A32Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[070f478](https://github.com/OfficeDev/office-js-docs-pr/commits/070f478175a2ee6402f5cb3b13ac0c8ab0702592)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/concepts/privacy-and-security?branch=pr-en-us-2129) |
[docs/outlook/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-nsd/docs/outlook/privacy-and-security.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/outlook/privacy-and-security?branch=pr-en-us-2129) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C9%5C14%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202009141645379384-2129%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=jeIrDquU%2BOQXR%2BAN1S%2Ba1o5AkY3qNcKwbJ6hEpPbdvk%3D&st=2020-09-14T16%3A41%3A36Z&se=2020-10-15T16%3A46%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**